hack/verify-bazel.sh

```diff
@@ -18,6 +18,8 @@ set -o nounset
 set -o pipefail
 set -o verbose
 
+if ! which bazel &>/dev/null; then exit; fi
```

Comment:
Should we print something here in case bazel isn't available?

Reply:
Done